lib/routes/security_pin/remote_server_auth.dart

```diff
@@ -361,15 +363,32 @@ class RemoteServerAuthPageState extends State<RemoteServerAuthPage> {
   }
 
   Future<DiscoverResult> testAuthData(RemoteServerAuthData authData) async {
-    log.info('remote_server_auth.dart: testAuthData');
+    log.info('remote_server_auth.dart: testAuthData: authData: $authData');
     try {
-      await widget._backupBloc
-          .testAuth(BackupSettings.remoteServerBackupProvider(), authData);
+      if (widget._torBloc.torConfig == null) {
```

Comment:
@erdemyerebasmaz  I don't understand this change. Why the testAuth is not good for non-tor users?